packages/components/src/data-model/word-query/error/error-mapper.js

```diff
@@ -0,0 +1,22 @@
+import WordQueryError from '@comp/data-model/word-query/error/word-query-error.js'
+
+/**
+ * A class that maps errors from one format to the other.
+ */
+export default class ErrorMapper {
+  /**
+   * Converts an error returned by client adapters to a format compatible with GraphQL schema.
+   *
+   * @param {object} clientAdaptersError - An error returned by client adapters.
```

Comment:
minor question: would this only ever be used for clientAdapter errors?  If we can envision other uses for it maybe the parameter should be more generally named.

Reply:
If we would take other error types as an argument, the method would require a different parameter set as different errors may have different error information in them and we have to supply the missing information with the parameters. I was thinking about this method as of something to convert an `AdapterError` to a `WordQueryError`. Maybe the method should be called like `adapterToWordQueryError`? How do you think it would be better to handle this?

Reply:
I think it's not that critical but if it's only to be used for mapping client adapter errors then I think it makes sense to make the class name reflect that.